cmd/cloud-controller-manager/app/options/options.go

```diff
@@ -58,6 +60,18 @@ func NewCloudControllerManagerOptions() *CloudControllerManagerOptions {
 	return &s
 }
 
+// NewDefaultCloudControllerManagerComponentConfig returns cloud-controller manager configuration object.
+func NewDefaultCloudControllerManagerComponentConfig(insecurePort int32) componentconfig.CloudControllerManagerConfiguration {
+	scheme := runtime.NewScheme()
+	componentconfigv1alpha1.AddToScheme(scheme)
```

Comment:
don't we need also the internal types here?

Reply:
little knowledge about it. just copy origin code from `controller-manager`'s `option.go`. @dims WDYT?

Reply:
I think there was a concern about polluting the existing  default scheme in the location where we were doing this  (and other internal types were not needed too)

Reply:
thank for your explaination